.mailmap

```diff
@@ -589,6 +589,8 @@ Gilbert Gede <gilbertgede@gmail.com> <ggede@ucdavis.edu>
 Gilles Schintgen <gschintgen@hambier.lu>
 Gina <Dr-G@users.noreply.github.com>
 Gleb Siroki <g.shiroki@gmail.com>
+Glenn Horton-Smith <glenn.hortonsmith@gmail.com>
+Glenn Horton-Smith <glenn.hortonsmith@gmail.com> Glenn Horton-Smith <gahs@phys.ksu.edu>
```

Comment:
I asked about this here: https://github.com/sympy/sympy/pull/23786#discussion_r928309963

I'll wait to hear back but otherwise this PR is good to go.

Reply:
Just wanted to confirm that this is good to go. Is there anything left for me to do to enable that?